examples/wrap_to_coroutine.py

```diff
@@ -15,15 +15,22 @@ def sync_snek():
 
 @wrap_to_coroutine
 async def async_snek():
-    asyncio.sleep(0.)
+    await asyncio.sleep(0.)
```

Comment:
I see

Thats how comments in code work... how about waiting longer than 0? I know its not necessary for your point, but this way, every programmer has the urge to remove this statement